src/components/routes/RouteInputDistance.vue

```diff
@@ -133,7 +133,7 @@ export default defineComponent({
             outlined
             type="number"
             v-model="distance"
-            min="1"
+            min="0"
```

Comment:
Mask example https://github.com/quasarframework/quasar/blob/dev/docs/src/examples/QInput/MaskFillReverse.vue

Reply:
See https://quasar.dev/vue-components/input#mask
Mask is only available if the type is one of ‘text’ (default), ‘search’, ‘url’, ‘tel’, or ‘password’.
I would prefer `number` type in this case. Do you think we should use input `text` + mask?

Reply:
> See https://quasar.dev/vue-components/input#mask Mask is only available if the type is one of ‘text’ (default), ‘search’, ‘url’, ‘tel’, or ‘password’. I would prefer `number` type in this case. Do you think we should use input `text` + mask?

I think yes this example work as expected.

Reply:
Make it in the new PR, please.